### PR TITLE
Styling changes and moving to templates

### PR DESCRIPTION
**Pull request by asengupta3**

### CODE REVIEW

**Comment by Zio-4** [comment]
@asengupta3 Is this ready for review? If not could you convert it to a draft? I didn't realize we could do that but we should use that feature.

File: task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js [hunk 1]
```diff
@@ -22,26 +28,11 @@ export function stimulus(isPractice = false, stage, stimulusDuration, postTrialG
...
-                    </div>
-                </div>`;
+      return getLayoutTemplate(
+        null,
+        mediaAssets.images[jsPsych.timelineVariable('stimulus')],
+        jsPsych.timelineVariable('position') <= 0.5
+      );
     },
     stimulus_duration: stimulusDuration,
     post_trial_gap: isPractice? null : postTrialGap,
```
Comment (asengupta3) [inline task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js:38]: @anabonne the `stimulus_duration` is causing the container to disappear after sometime. Is that the expected behavior and the `post_trial_gap` is causing a flicker on the screen as well. @kachergis @Zio-4  thoughts?

Comment (kachergis) [reply task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js:38]: as discussed, good to not have the container disappear -- thank you!

**Review by kachergis — APPROVED** [review]
looks great!

File: task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js (hunk 1 above)
Comment (anabonne) [reply task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js]: Yep we will need to make only the heart or flower disappear. For now we can just remove any timeouts on the stimulus.

**Pull request merged**